front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -172,6 +172,10 @@ public boolean isHealthy() {
     return (System.currentTimeMillis() - lastRefreshedTime.get()) < getHealthMillis() && allItemsCache.get() != null;
   }
 
+  public long getHealthIntervalMillis() {
```

Comment:
this feels a little dirty because it ends up overriding the `ItemDao.getHealthIntervalMillis` even though it doesn't directly implement ItemDAO (its subclasses do)